frontend/src/container/ListAlertRules/index.tsx

```diff
@@ -13,9 +13,10 @@ import ListAlert from './ListAlert';
 function ListAlertRules(): JSX.Element {
 	const { t } = useTranslation('common');
 	const location = useLocation();
-	const { data, isError, isLoading, refetch, status } = useQuery('allAlerts', {
+	const { data, isError, isLoading, status } = useQuery('allAlerts', {
 		queryFn: getAll,
 		cacheTime: 0,
+		refetchInterval: 30000,
```

Comment:
@palashgdev: Can we consider any other approach than polling? We can rely on the clone API response to refetch the alerts.

Reply:
@YounixM earlier we were dependent over refetch method itself over not this is handled by react-query itself

Reply:
Sorry, didn't understand the comment.

Reply:
there is useInternal in `https://github.com/SigNoz/signoz/pull/4082/files#diff-c3f4ae491202b106350fa9a32c645177e941926d48505e9f4647d83e9c1961b1` over that we are refetching the alerts... directly handling custom logic ... used inbuilt params for our use case